src/api/urls.py

```diff
@@ -52,6 +54,9 @@
         CancelCurrentMatchView.as_view(),
         name="cancel_current_match",
     ),
+    # Survey URLs
+    path("surveys/", SurveysView.as_view(), name="surveys"),
+    path("surveys/<int:id>/", SurveyView.as_view(), name="survey"),
```

Comment:
Should these be `/feedback/` or match_feedback or something since we're only calling them surveys internally? (This isn't a petty question, if you think we should just call them surveys from now on I'm 100% ok with that)

Reply:
I think this is a valid point, I'll change the route since surveys will probably confuse devs if we're referring to it as feedback.